### PR TITLE
III-3570 Fix eventStatus projection

### PR DESCRIPTION
**Pull request by bertramakers**
### Fixed
 
- Added `https://schema.org/` suffix to `eventStatus` projection on `Event` level
- Added `https://schema.org/` suffix to `eventStatus` projection on `subEvent` level
- Omitted `eventStatusReason` when empty
 
---
Ticket: https://jira.uitdatabank.be/browse/III-3570

### CODE REVIEW

**Comment by bertramakers** [comment]
@jdrieghe This PR is ready for review, unless you want to also include the fix for `eventStatus` being projected on Places in here as well?

File: src/Timestamp.php [hunk 1]
```diff
@@ -85,10 +85,14 @@ public function serialize(): array
 
     public function toJsonLd(): array
     {
-        $jsonLd = $this->serialize();
-        $jsonLd['@type'] = 'Event';
-
-        return $jsonLd;
+        return array_merge(
```
Comment (LucWollants) [inline src/Timestamp.php:88]: This is indeed a safer approach then reusing the `serialize` method.

**Review by LucWollants — APPROVED** [review]

File: src/Timestamp.php (hunk 1 above)
Comment (bertramakers) [reply src/Timestamp.php:88]: Yeah I don't mind reusing it if it's the same anyway, but when they start to diverge like now we should split up the serialization & json encoding. It's usually easy enough to start with the same and then split up later though so there was no harm in re-using `serialize()` at first.

**Comment by bertramakers** [comment]
@jdrieghe @LucWollants I'm going to close this PR since it focused mostly on adding the `https://schema.org/` prefix, which we now decided to leave out after all. The omission of the `eventStatusReason` can easily be fixed again later, when we also rename it to the new `statusReason` name like we discussed.

**Pull request closed without merging**